logstash-core/locales/en.yml

```
@@ -188,6 +188,8 @@ en:
           Duplicate keys found in your configuration: [%{keys}]
           At line: %{line}, column %{column} (byte %{byte})
           after %{after}
+        invalid_plugin_settings_multiple_codecs: |-
+          The %{type} plugin %{plugin} at line %{line} has multiple codecs configured - this is not a valid configuration, only one codec per plugin is supported.
```

We do have plugins that support multiple codecs (such as the http input's `additional_codecs` map), so the following is a slight re-wording to make it clear that we are talking about the top-level `codec` option, not _codecs_ as an abstract concept.

```suggestion
          The %{type} plugin %{plugin} at line %{line} has multiple `codec` declarations - this is not a valid configuration, only one top-level `codec` per %{type} plugin is supported.
```

---

That's great! thank you!